hack/make-rules/test-e2e-node.sh

```diff
@@ -66,7 +66,7 @@ if [ "${remote}" = true ] && [ -n "${debug_tool}" ]; then
 fi
 
 # Parse the flags to pass to ginkgo
-ginkgoflags="-timeout=24h"
+ginkgoflags=${GINKGO_FLAGS:-"-timeout=24h"}
```

Comment:
Seems reasonable to me, but I lack context. Which jobs invoke this `hack/make-rules/test-e2e-node.sh` and how?

Reply:
This is invoked by the `kubetest2` node tester. 

The Way `kubetest2` works is invoking this `test-e2e-node.sh` [here](https://github.com/kubernetes-sigs/kubetest2/blob/b64611bb4b03aab9dd9025a61a84385fec6fa557/pkg/testers/node/node.go#L281)

At the moment we have the kubetest2 versions of DRA skipping all tests because of this.

if this get merged, kubetest2 [PR](https://github.com/kubernetes-sigs/kubetest2/pull/286) introduces the usage of this `GINKGO_FLAGS`, and this [PR](https://github.com/kubernetes/test-infra/pull/33948) implements the usage for the DRA jobs in test-infra

Reply:
At the moment the only place where I can see this behavior is for the DRA kubetest2 jobs, but as we move more jobs to kubetest2 being able to pass ginkgo flags is a necessary feature

Reply:
Oh, no doubt :smile:

I'm just trying to understand the overall flow. See https://github.com/kubernetes-sigs/kubetest2/pull/286/files#r1884992400.